### ISSUE TITLE
Statistics

### ISSUE THREAD

**Issue by qaisjp** [chillroom/jacr-bot#15]
Here's a list of statistics that we tracked on plug.dj:

¹ ratings aren't currently being tracked. perhaps the website should show a question mark next to these sections, with a tooltip showing "these stats have been tracked since July 2016"
- total woots¹
- total grabs¹
- total mehs¹
- entire history of tracks that are played

With these stats, extra information is derived:
- average length
- total DJs (tracks played by a unique dj)
- total tracks played
- total time spent playing music ("time played")
- pie chart of woots against mehs
- pie chart of tracks that have only been played once, against tracks played multiple times
- tracks played only once / dj's djing only once (**duplicate of pie chart**)
- newest introduced artist (derived from track name)
- newest introduced track (never seen before)
- most woots/grabs on track/mix¹ (so four separate stats here)
- overall number of mixes
- highest room count
- tracking since

If you have any ideas for more visual statistics, make your suggestions below!
